### ISSUE TITLE
QSV not available from AppImage

### ISSUE THREAD

**Issue by alatteri** [CESNET/UltraGrid#270]
```
./UltraGrid-continuous-x86_64.AppImage --tool uv -d decklink -c libavcodec:help
H.265 - available (encoders: libx265 libsvt_hevc hevc_nvenc hevc_v4l2m2m hevc_vaapi, decoders: hevc hevc_v4l2m2m libde265 hevc_cuvid)

```

self compiled binary
```
 uv -d decklink -c libavcodec:help
H.265 - available (encoders: libx265 hevc_nvenc hevc_qsv hevc_v4l2m2m hevc_vaapi, decoders: hevc hevc_qsv hevc_v4l2m2m hevc_cuvid)
```

**Comment by MartinPulec** [comment]
I've already tried, please see #165 (in particular [this comment](https://github.com/CESNET/UltraGrid/issues/165#issuecomment-879084250)). The actual code is here in the branch [https://github.com/CESNET/UltraGrid/tree/wip-mfx](wip-mfx).

To sum up, it is quite complicated. I don't know current state, but the last version of Intel Media SDK that has been usable on U18.04 (we are currently building with U18.04) was not able to create a portable (working) binary. After we switch to U20.04 (after 4/2023), I can look at it again, but I've also done so back then when trying as well and it was also not working. So do not expect much, even after those 5 months - I assume that is rather less likely that it will work than vice versa.

**Comment by alatteri** [comment]
OK...no problem.  I can use self compile.  I didn't know it was a know issue.  Also, isn't FFMPEG going to be supporting OneVPL shortly??  Which should supercede MFX?

**Comment by MartinPulec** [comment]
> Also, isn't FFMPEG going to be supporting OneVPL shortly?? Which should supercede MFX?

I didn't know about that (just heard the name but doesn't check thoroughly). Yes, it does seem promising, I think that we may try it as soon as reaches upstream.

**Comment by alatteri** [comment]
According to this, it should already be in FFMPEG master.

https://www.phoronix.com/news/FFmpeg-Intel-oneVPL

**Comment by MartinPulec** [comment]
I've added support for openVPL should be to the AppImage. However, I don't have much time to test.

**Comment by alatteri** [comment]
I'll try to check it out over the weekend.

**Comment by MartinPulec** [comment]
I've finally get to try this one today - there was a dumb error but after the fix it works in my setup (Ubuntu 22.04 - it complained a bit about unsupported features but worked). It might work in [continuous](https://github.com/CESNET/UltraGrid/releases/tag/continuous) build, in stable channel next bug-fix release will be needed.

**Comment by alatteri** [comment]
I'm going to close this as it is superseded by the recent work Martin has been doing for QSV on NUC12.
